client/src/components/Contact/Contact.js

```diff
@@ -1,62 +1,124 @@
-import React from "react";
+import React, { useState } from "react";
 import "../../css/Contact.css";
+import { set } from "mongoose";
 
 const Contact = () => {
+
+  const [newContact, setNewContact] = useState({
+    name: "",
+    email: "",
+    website: "",
+    comment: ""
+  })
+  
+
+  const handleContact = (e) => {
+    const addContact = {
+      ...newContact,
+      [e.target.name]: e.target.value,
+    };
+
+    setNewContact(addContact);
+  };
+
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    fetch("http://localhost:5000/api/contact",
```

Comment:
alaways import domain and fetch it, cause this is not going to work on production